### DISCUSSION TITLE
Cannot push package to GitHub Container Registry

### DISCUSSION THREAD

**Discussion by mac-lucky** [community/community#45440] [Actions] [answered]
### Select Topic Area

Bug

### Body

The Error `denied: permission_denied: create_package`

I've got PAT to authenticate to ghcr.io. Thc scope is:
<img width="795" alt="obraz" src="https://user-images.githubusercontent.com/44085390/214876263-afb848de-1844-4d51-948e-5ffcb19dcec3.png">

Using `docker login ghcr.io` I logged in and got `Login success`

I created a simple script to build a package:

```
docker build -t maclucky/my-app .
docker tag maclucky/my-app:latest ghcr.io/maclucky/my-app:1.0.0
docker push ghcr.io/maclucky/my-app:1.0.0
```

than i have the error:
`denied: permission_denied: create_package`
and cannot push to the repo. 

What am I doing wrong?

**Comment by Zenexer** [comment, marked as answer]
When you logged in using `docker login ghcr.io`, did you enter your PAT as the password?

**Reply by mac-lucky** [reply]
I just noticed that my username was wrong:
`maclucky` and should be `mac-lucky`
